src/arviz_plots/plots/compareplot.py

```diff
@@ -0,0 +1,111 @@
+"""Compare plot code."""
+from arviz_base import rcParams
+
+from arviz_plots.visuals import (
+    labelled_title,
+    labelled_x,
+    labelled_y,
+    line_x,
+    line_y,
+    scatter_x,
+    yticks,
+)
+
+
+def plot_compare(cmp_df, color="black", target=None, backend=None):
+    r"""Summary plot for model comparison.
+
+    Models are compared based on their expected log pointwise predictive density (ELPD).
+
+    Notes
+    -----
+    The ELPD is estimated either by Pareto smoothed importance sampling leave-one-out
+    cross-validation (LOO) or using the widely applicable information criterion (WAIC).
+    We recommend LOO in line with the work presented by [1]_.
+
+    Parameters
+    ----------
+    comp_df : pandas.DataFrame
+        Result of the :func:`arviz.compare` method.
+    color : str, optional
+        Color for the plot elements. Defaults to "black".
+    similar_band : bool, optional
+        If True, a band is drawn to indicate models with similar
+        predictive performance to the best model. Defaults to True.
+    relative_scale : bool, optiona.
+        If True scale the ELPD values relative to the best model.
+        Defaults to True???
+    figsize : (float, float), optional
+        If `None`, size is (6, num of models) inches.
+    target : bokeh figure, matplotlib axes, or plotly figure optional
+    backend : {"bokeh", "matplotlib", "plotly"}
+        Select plotting backend. Defaults to rcParams["plot.backend"].
+
+    Returns
+    -------
+    axes :bokeh figure, matplotlib axes or plotly figure
+
+    See Also
+    --------
+    plot_elpd : Plot pointwise elpd differences between two or more models.
+    compare : Compare models based on PSIS-LOO loo or WAIC waic cross-validation.
+    loo : Compute Pareto-smoothed importance sampling leave-one-out cross-validation (PSIS-LOO-CV).
+    waic : Compute the widely applicable information criterion.
+
+    References
+    ----------
+    .. [1] Vehtari et al. (2016). Practical Bayesian model evaluation using leave-one-out
+       cross-validation and WAIC https://arxiv.org/abs/1507.04544
+    """
+    if backend is None:
+        backend = rcParams["plot.backend"]
+
+    # Maybe all this should be in a separate function
+    # that also checks what backends are supported and if they can be imported
+    if backend == "bokeh":
+        from bokeh.plotting import figure
+
+        target = figure()
+    elif backend == "matplotlib":
+        import matplotlib.pyplot as plt
+
+        _, target = plt.subplots()
+    elif backend == "plotly":
+        import plotly.graph_objects as go
+
+        target = go.Figure()
```

Comment:
Not sure this works for bokeh or plotly

Reply:
I would use `import_module` to import the corresponding plotting backend submodule, then use `create_plotting_grid(1)` (maybe we could also make `number=1` default so it can be called with `create_plotting_grid()` to generate a single plot. https://github.com/arviz-devs/arviz-plots/blob/main/src/arviz_plots/backend/none/__init__.py#L103. I would avoid backend dependent ifs as much as possible because then adding new backends won't automatically add support for that plot its source code needs to be modified. For example, this one would not work with "none" backend currently, it is not very important one, but it can be useful and we use it extensively for testing as it is much faster than the other ones given it doesn't plot anything.

For the rest I think it would also be easier to use `plot_bknd.blah` instead of using visuals as I don't think you actually need the extra logic in visuals which is mostly about dataarray processing. https://arviz-plots.readthedocs.io/en/latest/gallery/plot_forest_pp_obs.html uses the import and plot_backend.xlabel so it can serve as an example.